CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 3.0.0
```

Comment:
Is this intentional? Version 3.0.0 will come with a React upgrade and therefore may be better as a Minor version. @andrewjtait

Reply:
@adrianpastur was unsure how quickly they need the fix, so just targeting the next planned release unless they decide otherwise

Reply:
We have had no reports from customers about this issue so there is no need for a quick release.